packages/frontend/src/main/pages/stream/CommitObjectViewer.vue

```diff
@@ -1,15 +1,22 @@
 <template>
   <div class="commit-object-viewer">
-    <div v-if="(isMultiple || isCommit || isObject) && !singleResourceError">
+    <div
```

Comment:
The changes you see in seemingly unrelated Vue files are from when I was testing out Volar and fixing TS errors I found in some of the templates